gce/survival-training/dummy/__init__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2017 Google Inc. All Rights Reserved.
+#
```

Comment:
Why are these files under dummy dir?

Reply:
It's a dummy trainer script. Is your comment about the name or the existence of such a script?

The plan for sklearn is to wrap up sklearn training inside the dummy script, so I could call it `wrapper` instead.

Reply:
The comment was about calling it "dummy".

Reply:
Changed to `wrapper`